### PR TITLE
[tests-only] Remove passing tests from pending state (1)

### PR DESCRIPTION
**Pull request by saw-jan**
Remove passing tests out of pending state

### Related issue
Part of https://github.com/owncloud/owncloud-sdk/issues/1089

### CODE REVIEW

**Comment by saw-jan** [comment]
Blocked until https://github.com/owncloud/owncloud-sdk/pull/1085

File: tests/helpers/pactHelper.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const applicationXmlResponseHeaders = {
   'Content-Type': 'application/xml; charset=utf-8'
 }
 const textPlainResponseHeaders = {
-  'Content-Type': 'text/plain; charset=utf-8'
+  'Content-Type': MatchersV3.regex('text/plain(;()?charset=(utf|UTF)-8)?', 'text/plain; charset=utf-8')
```
Comment (saw-jan) [inline tests/helpers/pactHelper.js:73]: Can we do loose match here or not acceptable?

Matches:
`text/plain; charset=utf-8`
`text/plain;charset=utf-8`
`text/plain; charset=UTF-8`
`text/plain`

File: tests/helpers/pactHelper.js [hunk 2]
```diff
@@ -208,7 +208,7 @@ const deleteResourceInteraction = (
   if (resource.includes('nonExistent')) {
     response = {
       status: 404,
-      headers: applicationXmlResponseHeaders,
+      headers: xmlResponseHeaders,
```
Comment (saw-jan) [inline tests/helpers/pactHelper.js:211]: Also, can we treat `application/xml` and `text/xml` as same?

Comment (individual-it) [reply tests/helpers/pactHelper.js:211]: It think yes, see https://github.com/owncloud/ocis/issues/1686

File: tests/helpers/pactHelper.js (hunk 1 above)
Comment (saw-jan) [reply tests/helpers/pactHelper.js:73]: I will collect the data

File: tests/capabilitiesTest.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ describe('Main: Currently testing getConfig, getVersion and getCapabilities', ()
   } = require('./helpers/pactHelper.js')
 
   it('checking method : getConfig', async () => {
-    const provider = createProvider(false, true)
+    const provider = createProvider(false, false)
```
Comment (individual-it) [inline tests/capabilitiesTest.js:17]:
```suggestion
    const provider = createProvider()
```
as `false` is the default we could get rid of the parameters where-ever its false for both

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**